python/lsst/hsm_moments.py

```diff
@@ -0,0 +1,111 @@
+# This file is part of meas_extensions_shapeHSM.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import galsim
+import lsst.pex.config as pexConfig
+from lsst.meas.base import SingleFramePluginConfig, SingleFramePlugin
+# import MeasurementError
+
+class HsmMomentsConfig(SingleFramePluginConfig):
+    # roundMoments
+    # addFlux
+    roundMoments = pexConfig.Field[bool](doc="Use round weight function?", default=False)
+    addFlux = pexConfig.Field[bool](doc="Store measured flux?", default=False)
+
+
+class HsmSourceMomentsConfig(HsmMomentsConfig):
```

Comment:
You need to add the following decorator to this function in the line above
```python
@lsst.meas.base.register("ext_shapeHSM_HsmSourceMoments")
```

and remove these lines: https://github.com/lsst/meas_extensions_shapeHSM/blob/main/python/lsst/meas/extensions/shapeHSM/__init__.py#L40-L41

Similarly, you should add decorators to register the various plugins and keep the same names as used in the `__init__`. The removal of lines as you add decorators is to avoid any naming clashes.